### DISCUSSION TITLE
Multiple separate bundles with a single run?

### DISCUSSION THREAD

**Discussion by alberto-schena-gizero** [parcel-bundler/parcel#7304] [Q&A]
Hello,
I have an application consisting of multiple dashboards to be bundled separately.

Is it possible to bundle them with a single run of build/watch APIs?
Both build and watch modes are needed to be working.

In addition, the dashboard list is to be constructed dynamically.
Thus, Parcel should not rely on static configuration entries (e.g. targets?) within package.json.

Also, the reporter should indicate the bundle name in progress and error messages.
Is there a reporter suitable for multi-bundle, possibly parallel, usage?

Would you please give a code example for achieving these requirements?

Thank you in advance for your attention and reply!

Kind regards,
Alberto Schena

**Comment by mischnic** [comment]
Your options are using targets, globs (`parcel build src/*.js`), or the [API](https://parceljs.org/features/parcel-api/) to generate a list of targets or entry points programatically.
